src/components/ProductItem/ProductItem.tsx

```diff
@@ -38,6 +38,7 @@ export interface ProductProps {
   item: Product;
   currencySymbol: string;
   currencyRate?: string;
+  categoryConfig?: Record<string, any>;
```

Comment:
Should we define a proper object, where new properties could added explicitly as they are needed ?

Reply:
This is configurable through excel sheet.
IMO It won't be worth the effort